external/storm-redis/src/main/java/org/apache/storm/redis/bolt/RedisFilterBolt.java

```diff
@@ -45,18 +48,41 @@
  */
 public class RedisFilterBolt extends AbstractRedisBolt {
     private final RedisFilterMapper filterMapper;
+    private final StreamMapper streamMapper;
     private final RedisDataTypeDescription.RedisDataType dataType;
     private final String additionalKey;
 
     /**
-     * Constructor for single Redis environment (JedisPool)
+     * Constructor for single Redis environment (JedisPool).
+     * Tuples will be emitted to Storm's default streamId.
      * @param config configuration for initializing JedisPool
      * @param filterMapper mapper containing which datatype, query key that Bolt uses
      */
     public RedisFilterBolt(JedisPoolConfig config, RedisFilterMapper filterMapper) {
```

Comment:
Do you want to target this for master?  Or do you also want this in 1.x?  If it is just master we can play some games with a default method implementation in the FilterMapper interface.

If you want it in 1.x I would suggest that we leave FilterMapper untouched and create a LookupMapper that also has the same, or similar methods to FilterMapper, but is not a FilterMapper.  Then you can have a wrapper class that is a LookupMapper, but takes a FilterMapper.  The code could then wrap any FilterMapper passed in, and just use the LookupMapper interface.

I prefer the first one with the default methods because it reduce the number of classes and interfaces but is also binary compatible.  If we are not on java 8 like 1.x then we cannot use default methods.